src/unraphael/dash/pages/4_compare.py

```diff
@@ -181,50 +224,89 @@ def alignment_help_widget():
     )
 
 
+# def animate_images(img_aligned, baseline, num_frames=100):
+#     fig, ax = plt.subplots()
+#     blended_image = cv2.addWeighted(img_aligned, 1, baseline, 0, 0)
+#     im = ax.imshow(cv2.cvtColor(blended_image, cv2.COLOR_BGR2RGB), extent=[0, blended_image.shape[1], 0, blended_image.shape[0]])
+
+#     def update(frame):
+#         alpha = frame / num_frames
+#         blended_image = cv2.addWeighted(img_aligned, 1 - alpha, baseline, alpha, 0)
+#         im.set_array(cv2.cvtColor(blended_image, cv2.COLOR_BGR2RGB))
+#         ax.set_title(f'Frame {frame + 1}/{num_frames}')
+                           
+#     ani = animation.FuncAnimation(fig, update, frames=num_frames, interval=25, repeat=True, repeat_delay=1000)
+#     return ani.to_jshtml()
+                
+
 def comparison_widget(
     base_image: ImageType,
     images: list[ImageType],
 ):
     """Widget to compare processed images."""
     st.subheader('Comparison')
 
-    col1, col2 = st.columns((0.3, 0.7))
+    col1, col2 = st.columns((0.20, 0.80))    
+
+    if 'count_comp' not in st.session_state:
+        st.session_state.count_comp = 0
+
+    def display_image():
```

Comment:
Is this function duplicated from above?

This does not seem to belong here.

Reply:
Then I don't understand what this code is supposed to do I guess. 

> This PR adds a 'previous' and 'next' button ... It is intended as an alternative to the slider comparison.

It seems to me that both the slider and the side-by-side widget now have these buttons.

Reply:
@stefsmeets  I refactored the code to remove redundancy in the code surrounding the display of images. Now the comparison and images alongside each other are integrated. Finally, indeed, both the slider and the side-by-side option  have these previous and next buttons; works like a charm!